src/warc2zim/replay/topFrame.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+html, body
+{
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+}
+
+iframe {
+  width: 100%;
+  height: 100%;
+}
+
+</style>
+</head>
+<body style="margin: 0; padding: 0;">
+<iframe id="replay_iframe" frameborder="0" seamless="seamless" scrolling="yes" class="wb_iframe" allow="autoplay; fullscreen"></iframe>
+<script>
+  const PREFIX = "$PREFIX";
+  const URL = "$URL";
+
+  const iframe = document.querySelector("iframe");
+
+  // update URL when iframe changes
+  window.addEventListener("message", function() {
+    if (event.data.wb_type === "load" || event.data.wb_type === "replace-url") {
+      
+      // remove scheme to be consistent with current canonicalization
+      const urlNoScheme = event.data.url.slice(event.data.url.indexOf("//") + 2);
+      window.history.replaceState(null, "", PREFIX + urlNoScheme);
+    }
+  });
+
+  iframe.src = PREFIX + "mp_/" + URL;
```

Comment:
Is `mp_` a sort of namespace?  Where does this come from? sw.js?

Reply:
Yes, sort of, it is a part of the replay.. Probably needs better documentation in wabac.js
I have it documented in pywb, which is the python version of the rewriting system, but it is the same idea:
https://pywb.readthedocs.io/en/latest/manual/rewriter.html#main-page-modifier-mp

Without the modifier, it serves the outer frame.
With the `mp_/` modifier it serves the rewritten content.